svc-bip-api/src/main/java/gov/va/vro/bip/config/ConfigForGetClaimDetails.java

```diff
@@ -0,0 +1,41 @@
+package gov.va.vro.bip.config;
+
+import gov.va.vro.bip.model.BipClaim;
+import org.springframework.amqp.core.*;
+import org.springframework.amqp.rabbit.listener.api.RabbitListenerErrorHandler;
+import org.springframework.amqp.rabbit.support.ListenerExecutionFailedException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ConfigForGetClaimDetails {
+    @Autowired
+    DirectExchange bipApiExchange;
+    @Bean
+    Queue getClaimDetailsQueue() {
+        return new Queue("getClaimDetailsQueue", true, false, true);
```

Comment:
There are reasons why I did it this way.  would you mind stating why I should change that?

Reply:
This value should be assigned with the property because it makes changing the name of the queue easier when it is used in multiple places. Spring will inject whatever value is set in the application properties everywhere where the field or parameter is annotated with `@Value`. Also, this property is already being injected in the RMQ integration test. 

You said you had reasons? I'd like to hear them!

Reply:
you are reviewing some outdated version.  This was already [changed to follow your advice ](https://github.com/department-of-veterans-affairs/abd-vro/blob/e51a9a97a15623db105728c8c15b992fc0728e99/svc-bip-api/src/main/java/gov/va/vro/bip/config/ConfigForGetClaimDetails.java#L15).  

The reasons against it is simplicity.  Using @Value is over-engineering for a value that'll never change.  To achieve it, you still end up copy pasting strings that are not compiler checked a bunch of times.